config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :doctors, only: [:show]
+  resources :doctor_patients, only: [:destroy]
```

Comment:
This route _can_ be restful, but one thing to keep in mind that this route will be something like this: `"/doctor_patients/:id"`.